jenkins/cleanup_python_code.py

```diff
@@ -0,0 +1,46 @@
+"""
+Script to help create a PR. To be run by
```

Comment:
This looks good to me. Only one possible suggestion:

As far as I see, this file and jenkins/upgrade_python_requirements.py do pretty much the same. I think that the only differences are the pr_body and the parameters used in PullRequestCreator and delete_old_pull_requests. 

I think that we can keep only one of those files using a more generic name, add parameters for the  things that differ in both files and add the value of the parameters in each resouce of jenkins-job-dsl

For instance, testeng/resources/cleanup-python-code-create-pr in
https://github.com/edx/jenkins-job-dsl/pull/1040/files

But as I said, it is only a suggestion, I think that we can merge this as is.

Reply:
good point, but as you can see we've added some extra arguments that'll only be utilized by `cleanup-python-code` job